devicetypes/smartthings/zll-rgb-bulb.src/zll-rgb-bulb.groovy

```diff
@@ -101,6 +101,8 @@ def refresh() {
 }
 
 def poll() {
+	configureHealthCheck()
```

Comment:
It should only get called once if the "custom health check" hasn't been setup. I added it to poll to account for existing devices with the DTH already setup, since deploy (from what I recall) won't call update automagically.